raisinbread/RB_files/RB_parameter_type.sql

```diff
@@ -12,5 +12,188 @@ INSERT INTO `parameter_type` (`ParameterTypeID`, `Name`, `Type`, `Description`,
 INSERT INTO `parameter_type` (`ParameterTypeID`, `Name`, `Type`, `Description`, `RangeMin`, `RangeMax`, `SourceField`, `SourceFrom`, `SourceCondition`, `Queryable`, `IsFile`) VALUES (10,'candidate_label','text','Identifier_of_candidate',NULL,NULL,'PSCID','candidate',NULL,1,0);
```

Comment:
I can't comment on the history file above because the diff is too large.. but is adding the whole history table to RB really necessary if it wasn't there before?

Reply:
I was also wondering about this. I don't think we need it.

Reply:
I usually dont commit the history changes
it happened accidentally here since it got modified by the data_dictionary_builder

I dont see the issue with it, but i can remove it if need be